pandas/core/generic.py

```diff
@@ -3749,8 +3752,13 @@ class   animal   locomotion
             )
 
         else:
```

Comment:
can you make this part of the higher level if/else

Reply:
Done